defaults/main.yml

```diff
@@ -2,9 +2,10 @@
 # WARNING: This may not be your desired MGMT address.
 nova_my_ip: '{{ ansible_default_ipv4.address }}'
 # RabbitMQ
-rabbit_hostname: 'localhost'
-rabbit_username: 'rabbit_username_default'
-rabbit_password: 'rabbit_pass_default'
+openstack_nova_compute_rabbit_hostname: 'localhost'
+openstack_nova_compute_rabbit_username: 'rabbit_username_default'
+openstack_nova_compute_rabbit_password: 'rabbit_pass_default'
```

Comment:
I'm not really sure if assigning a default username and password is a good thing.
I'd prefer we don't assign any default and force users to specify one.
What do you think?